Docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine:3.17 AS build-env
+
+RUN apk add --update --no-cache git
+
+RUN git clone --depth 1 https://github.com/dannydeezy/deezy-auto-earn /deezy-auto-earn
```

Comment:
is `depth` necessary here? what if the repo changes to add some subfolders, would that still work?

Reply:
Yes, it will basically check out only the most recent state (1 level of history) from the repository. This is not folder recursion.

> --depth <depth>
Create a shallow clone with a history truncated to the specified number of commits. Implies --single-branch unless --no-single-branch is given to fetch the histories near the tips of all branches. If you want to clone submodules shallowly, also pass --shallow-submodules.

https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt

Reply:
ah gotcha thank you